packages/feedback-service/src/feedback-config/typedef.graphql

```diff
@@ -16,7 +16,8 @@ input FeedbackHeaderInput {
 
 type FeedbackConfigType {
   id: ID
-  appId: String
+  appId: String @deprecated(reason: "appId is deprecated. Use projectId instead.")
```

Comment:
`appId` is not deprecated, it will be used as metadata to identify the SPA from SPAship, if needed.
```suggestion
  appId: String
```

Reply:
Can't we use projectId instead. That would be the single source of truth for

Reply:
Removed